src/Components/CloseIcon.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import closeIcon from '../MediaFiles/closeicon.svg';
+
+const CloseIcon = ({ onClick }) => {
+  const iconStyle = {
+    width: "20px",
+    height: "20px",
+    cursor: "pointer",
+    position: "absolute",
+    top: "30px",
+    right: "30px",
+    //zIndex: 1000, // Ensure it's on top of other elements
```

Comment:
In this case, zIndex would had taken effect only if there was another absolute positioned element. You don't need it here.